tlslite/extensions.py

```diff
@@ -529,6 +529,71 @@ def parse(self, p):
 
         return self
 
+
+class SupportedVersionsExtension(TLSExtension):
+    """
+    This class handles the SupportedVersion extensions used in TLS 1.3.
+
+    See draft-ietf-tls-tls13.
```

Comment:
maybe referencing the precise draft is more suitable. Although I also don't expect the format of that to change, other messages change over various drafts.